web/src/styles/dracula.css

```diff
@@ -14,7 +14,6 @@
   display: block;
   overflow-x: auto;
   padding: 0.5em;
-  background: #282a36;
```

Comment:
Removed this otherwise the codeblock had a padding of different background colour. Removing this means it was consistently black.